changesets2CSV/changesets2CSV

```diff
@@ -105,6 +105,17 @@ def create_weekly(args):
     args.end_time = str(end)
     create_summary(args)
 
+def count_new_modified_deleted(url):
+    api_url = "https://www.openstreetmap.org/api/0.6/changeset/{changeset}/download"
+    changeset = url.rpartition('/')[2]
```

Comment:
could we send the id from the dictionary instead of splicing up the url?

Reply:
I didn't see the ID field when I was looking at the fields -- I saw URL and used that. I've fixed that.